firebase_admin/ml.py

```diff
@@ -841,8 +839,12 @@ def handle_operation(self, operation, wait_for_operation=False, max_time_seconds
                 raise _utils.handle_operation_error(operation.get('error'))
             raise exceptions.UnknownError(message='Internal Error: Malformed Operation.')
 
-        op_name = operation.get('name')
-        _, model_id = _validate_and_parse_operation_name(op_name)
+        op_name = _validate_operation_name(operation.get('name'))
```

Comment:
Nit: Push this further down, as it's not immediately used.

Reply:
It's an early out. If the operation name is not valid, you won't be able to get_operation below so you might as well not bother with unpacking metadata or setting start and stop times etc. Also, pushing it down would make the testing a bit harder as we'd have to include valid metadata with the op_name validation tests.